### DISCUSSION TITLE
Mojo 0.6.1 and Modular CLI 0.3.1 released!

### DISCUSSION THREAD

**Discussion by jackos** [modular/modular#1519] [Announcements]
Modular 💻 0.3.1 and Mojo 🔥 0.6.1 has been released! 🚀

Please [check here](https://modul.ar/download) for instructions on how to upgrade or install

And check out the [changelog here](https://docs.modular.com/mojo/changelog.html#v0.6.1-2023-12-18)

Happy Holidays!

**Comment by ivellapillil** [comment]
Great. One concern on the CollectionElement. Once intersection types are implemented, would collections still insist on conformance to CollectionElement or would Collections then take intersections of Moveable, Copyable etc.?

**Comment by mojo-for-ai** [comment]
When will Mojo🔥 0.7 be released?

**Comment by 2005Ashwani** [comment]
when mojo will released

**Reply by Eprahim-taha** [reply]
Do you mean when will Mojo 🔥 be open source? 

The answer is in the first quarter of 2024
